.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+<!--  Sorry, client-go does not accept pull requests yet. Please submit your 
```

Comment:
Does it have to be in an html comment?

s/accept pull requests yet/accept changes via pull requests at this time/